tests/unit/test_protocol_errors.py

```diff
@@ -74,25 +66,22 @@ def assertRequestRaises(self, exceptionClass, url, request):
         corresponding to the specified exception class.
         """
         self.assertRawRequestRaises(
-            exceptionClass, url, request.toJsonString())
+            exceptionClass, url, protocol.toJson(request))
 
     def testPageSize(self):
         for url, requestClass in self.endPointMap.items():
-            for badType in ["", "1", "None", 0.0, 1e3]:
-                request = self._createInstance(requestClass)
-                request.pageSize = badType
-                self.assertRequestRaises(
-                    exceptions.RequestValidationFailureException, url, request)
-            for badSize in [-100, -1, 0]:
-                request = self._createInstance(requestClass)
-                request.pageSize = badSize
+            # FIXME: Can't test 0 because looks the same as unset to Protobuf
```

Comment:
Delete the FIXME: here --- this is just something we have to learn to live with.